src/Listener/JsonApiListener.php

```diff
@@ -329,32 +329,35 @@ protected function _fieldSetsParameter($fieldSets, Subject $subject, $options)
         $fieldSets = array_map(function ($val) {
             return explode(',', $val);
         }, $fieldSets);
-        if (empty($fieldSets)) {
-            $this->config('fieldSets', $fieldSets);
-        }
 
```

Comment:
This was a typo, intended to be:

```
if (!empty($fieldSets)) [
    $this->config('fieldSets', $fieldSets);
}
```

However, we call `$this->config('fieldSets', $fieldSets);`, it broke other things

Any insights on who/what/when to set this?